config/initializers/honeycomb.rb

```diff
@@ -0,0 +1,24 @@
+require 'libhoney'
```

Comment:
The code in this file was dumped verbatim from the "getting started" example.

Reply:
maybe drop a link to that example in a comment here?

Reply:
Great idea.   I also found every logging/profiling tool ever made for Rails by googling for this line from the example:
```
 ActiveSupport::Notifications.subscribe(/process_action.action_controller/) do
```
They all use it :)